components/form/form-element-mixin.js

```diff
@@ -123,9 +123,9 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		return this.localize('components.form-element-mixin.defaultFieldLabel');
 	}
 
-	async requestValidate() {
+	async requestValidate(showErrors) {
```

Comment:
I'm not a huge fan of `showErrors` so if anyone has any better names let me know or if we want this to default to `true` rather than `false`.

Reply:
Not sure on the name but I'm thinking we may want to default to `true` instead of `false` to avoid confusion around why errors aren't working (which may have just happened to me when I was trying this out and it took me a bit to figure out where I was missing a true)

Reply:
Yeah I'm not thinking of anything better. You could reverse it to be something like `suppressErrors` or `hideErrors`... not sure that's an improvement. `displayErrors`, `visualizeErrors`, `errorsAreShown`... haha.

Reply:
Sounds good to me, in that case I'll leave the name as is but make it default to `true`.

Reply:
Changed `showErrors` to default to `true` but only for `requestValidate()` since `validate()` should never be called manually. Only thing to watch out for is that you don't lose the `showErrors` value when you override `validate()`.